src/audio/tables/samplebank_table.c

```diff
@@ -0,0 +1,50 @@
+#include "attributes.h"
+#include "z64audio.h"
+
+// Symbol definition
+
+extern AudioTable gSampleBankTable;
+#pragma weak gSampleBankTable = sSampleBankTableHeader
+
+// Externs for table
+
+#define DEFINE_SAMPLE_BANK(name, medium, cachePolicy) \
+    extern u8 name##_Start[];                         \
+    extern u8 name##_Size[];
+#define DEFINE_SAMPLE_BANK_PTR(name, medium, cachePolicy) /*empty*/
```

Comment:
for all of these `#define DEFINE_SAMPLE_BANK_PTR`s, should the first parameter name be `index`?